projects/plugins/jetpack/_inc/client/components/product-activated/index.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const ProductActivated = ( { active } ) => {
+	// Don't display the label if product isn't active.
+	if ( active !== '1' ) {
+		return null;
+	}
+
```

Comment:
I think we can remove this check and the need to pass the `active` prop entirely if move the check to the point where the component is created (see the comment below).

Reply:
Thanks for the suggestion! I agree that it would simplify the component's logic a bit.